recipes/sdl2/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "SDL2" %}
+{% set version = "2.0.4" %}
+{% set sha256 = "da55e540bf6331824153805d58b590a29c39d2d506c6d02fa409aedeab21174b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn:  {{ name }}-{{ version }}.tar.gz  # [unix]
+  url:  https://www.libsdl.org/release/{{ name }}-{{ version }}.tar.gz  # [unix]
+  sha256: {{ sha256 }}  # [unix]
+  fn:  {{ name }}-{{ version }}.zip  # [win]
+  url:  https://www.libsdl.org/release/{{ name }}-{{ version }}.zip  # [win]
```

Comment:
Had to generate that myself, but totally forgot about it. Will add the sha256 (08ba1de8bb4236d3aa1a94168b6e1a4f286bd21ca3437c11689d4e55f3a9cc89) to the recipe.